lib/src/tab_controller.dart

```diff
@@ -0,0 +1,56 @@
+part of 'hooks.dart';
+
+/// Creates and disposes a [TabController].
+///
+/// See also:
+/// - [TabController]
+TabController useTabController({
```

Comment:
Keeping vsync optional here would be better.

Reply:
Good point! I fixed that